.github/workflows/ci.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, ubuntu-20.04]
-        elixir_version: [1.12.3, 1.13.3, 1.14.1]
-        otp_version: [24, 25]
+        elixir_version: [1.13.3, 1.14.1, 1.15]
```

Comment:
I'm removing 1.12 since most libraries support up to three most recent Elixir version.

Reply:
yea also there is a big deprecation around log warn so might as well move to minimum at 1.13